### PR TITLE
Added Slalom & Maintain Heading manifests & simulation course maps

### LINKED ISSUE #18: [FEATURE] Add maintain-heading manifest file
### Description
Add maintain heading manifest file. Use the same format as manifests/speed-test.json

### Estimated Effort
0.5

### Anything that could be affected by this? (N/A for nothing)
N/A

### LINKED ISSUE #19: [FEATURE] Add Slalom manifest file
### Description
Add Slalom manifest file. Use the same format as manifests/speed-test.json

### Estimated Effort
0.5

### Anything that could be affected by this? (N/A for nothing)
N/A

### LINKED ISSUE #21: [FEATURE] Add maintain heading simulation map
### Description
Add maintain heading simulation-map json file. Format as described in [the readme](https://github.com/UIOWA5830SP19/SPP100/blob/development/BoatOS/src/main/resources/simulation-maps/README.md)

### Estimated Effort
1

### LINKED ISSUE #22: [FEATURE] Add Slalom simulation map
### Description
Add slalom simulation-map json file. Format as described in [the readme](https://github.com/UIOWA5830SP19/SPP100/blob/development/BoatOS/src/main/resources/simulation-maps/README.md)

### Estimated Effort
1

### Anything that could be affected by this? (N/A for nothing)
N/A

### PR DESCRIPTION
**Pull request by cooperbell**
### What's in this pull request
* Added manifest file for the maintain heading test. Closes #18 
* Added manifest file for the slalom test. Closes #19 
* Added simulation course map for the maintain heading test. Closes #21
* Added simulation course map for the slalom test. Closes #22

### CODE REVIEW

File: BoatOS/src/main/resources/manifests/maintainHeading-test.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        },
+        {
+          "class": "com.imarc.marc_i.cv.CVNode",
+          "data": "src/main/resources/node-data.json"
+        }
+      ]
+    }
+  ],
+  "tasks": [
+    "onMaintainHeadingTest"
```
Comment (jwryan9) [inline BoatOS/src/main/resources/manifests/maintainHeading-test.json]: Will need to be sure we add methods associated to this task to relevent nodes. Will use speedTest task as reference.

File: BoatOS/src/main/resources/manifests/slalom-test.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        },
+        {
+          "class": "com.imarc.marc_i.cv.CVNode",
+          "data": "src/main/resources/node-data.json"
+        }
+      ]
+    }
+  ],
+  "tasks": [
+    "onSlalomTest"
```
Comment (jwryan9) [inline BoatOS/src/main/resources/manifests/slalom-test.json]: Same as above.

File: BoatOS/src/main/resources/manifests/maintain-heading-test.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        },
+        {
+          "class": "com.imarc.marc_i.cv.CVNode",
+          "data": "src/main/resources/node-data.json"
+        }
+      ]
+    }
+  ],
+  "tasks": [
+    "onMaintainHeadingTest"
```
Comment (jwryan9) [inline BoatOS/src/main/resources/manifests/maintain-heading-test.json]: Will need to be sure we add methods associated to this task to relevent nodes. Will use speedTest task as reference.

Comment (aklinker1) [reply BoatOS/src/main/resources/manifests/maintain-heading-test.json]: Not every node needs the task methods. If a node doesn't have it, it just skips it. So you only really have to include them in nodes they're relevant to.

Relevant code:
- [Task Loop](https://github.com/UIOWA5830SP19/SPP100/blob/8cc2efbc533a6192a5213ad84201f83d6ab36706/BoatOS/src/main/java/com/imarc/marc_i/arch/nodes/NodeManager.java#L142)
- [skipping non-existing task functions](https://github.com/UIOWA5830SP19/SPP100/blob/8cc2efbc533a6192a5213ad84201f83d6ab36706/BoatOS/src/main/java/com/imarc/marc_i/arch/nodes/Node.java#L95)

Just a heads up, you might have already known this.

File: BoatOS/src/main/resources/simulation-maps/maintain-heading-1-test-map.json [hunk 4]
```diff
@@ -0,0 +1,106 @@
+{
```
[the remark this answers is not in the document]
Comment (cooperbell) [reply BoatOS/src/main/resources/simulation-maps/maintain-heading-1-test-map.json]: Yeah, I'll do that in a later PR

**Pull request merged**